### PR TITLE
:arrow_up::one::one:, pos_multisession_menu

### PR DESCRIPTION
**Pull request by fedoranvar**

### CODE REVIEW

**Review by yelizariev — APPROVED** [review]
No code was changed

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
In index.rst file: "Tested  on Odoo 10.0 community". Please change to actual version and comment enterprise.

**Comment by fedoranvar** [comment]
> In index.rst file: "Tested on Odoo 10.0 community". Please change to actual version and comment enterprise.

check again, please

**Comment by Ramil-Mukhametzyanov** [comment]
> > In index.rst file: "Tested on Odoo 10.0 community". Please change to actual version and comment enterprise.
> 
> check again, please
Sorry, in index.html

**Review by Ramil-Mukhametzyanov — APPROVED** [review]

**Review by Ramil-Mukhametzyanov — COMMENTED** [review]
@fedoranvar Please add the module name to the PR title.

**Review by KolushovAlexandr — APPROVED** [review]

File: pos_multi_session_menu/README.rst [hunk 1]
```diff
@@ -30,12 +31,12 @@ Maintainers
...
 
-HTML Description: https://apps.odoo.com/apps/modules/10.0/pos_multi_session_menu/
+HTML Description: https://apps.odoo.com/apps/modules/11.0/pos_multi_session_menu/
 
 Usage instructions: `<doc/index.rst>`_
 
 Changelog: `<doc/changelog.rst>`_
 
-Tested on Odoo 10.0 0351677a00df8bd76fa207a38dc2ecdefdc6940a
+Tested on Odoo 11.0 dff2e242a5e5288cb3c8408a9b635a4c85034e83
```
Comment (em230418) [inline pos_multi_session_menu/README.rst]: Looks like it is from 12.0 branch
https://github.com/odoo/odoo/commit/dff2e242a5e5288cb3c8408a9b635a4c85034e83

**Review by em230418 — CHANGES_REQUESTED** [review]
Only commit hash change required.

**Review by em230418 — APPROVED** [review]

**Pull request merged**